katsdpcontroller/sensor_proxy.py

```diff
@@ -59,7 +61,7 @@ def __init__(self, client: aiokatcp.Client, server: aiokatcp.DeviceServer,
         self._interface_changed = False
 
     def rewrite_name(self, name: str) -> str:
-        return self.prefix + name
+        return self.renames.get(name, self.prefix + name)
```

Comment:
I understand that it will only ever receive a `sensor.name` for one that is correctly instantiated, but if I understand this correctly:
* This `get` call will return `self.prefix + name` whether the name is present or not.

Is that/should it be a concern?

![image](https://user-images.githubusercontent.com/25218138/145353532-f10b2b74-d6b9-44a0-aaf1-cc9b6ca69a4a.png)

Reply:
It'll return `self.renames[name]` if that exists, otherwise `self.prefix + name`. So anything without an explicit override will default to the old behaviour.